src/style.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: Avenir;
+  src: url(../src/fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: "Avenir" , sans-serif;
+}
+
+.search {
+  display: flex;
+  align-items: center;
+  margin: 20px auto;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  width: 100%;
+  border-radius: 4px;
+  box-sizing: border-box;
+  font-family: "Avenir" , sans-serif;
+}
+
+.big-search {
+  height: 70px;
+}
+
+.small-search {
+  height: 42px;
+}
+
+.big-img-search {
+  padding: 24px 15px 25px 25px;
+}
+
+.small-img-search {
+  width: 11px;
+  height: 11px;
+  vertical-align: middle;
+  padding: 14px 9px 15px 12px;
+}
+
+.big-input-search ,
+.small-input-search {
+  width: 100%;
+  border: none;
+}
+
+.big-input-search {
+  font-size: 16px;
+  padding-left: 3px;
+}
+
+.small-input-search {
+  font-size: 14px;
+  padding-left: 1px;
+}
+
+.big-input-search ,
+.small-input-search ::placeholder {
+  color: #3e4e61;
+}
+
+.search:hover {
+  border-radius: 4px;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus-within {
+  font-family: inherit;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: none;
+}
```

Comment:
```suggestion
.search:focus-within {
  font-family: inherit;
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
  outline: none;
}
```

Reply:
You forgot to disable outline again 🙂

Reply:
But now, when you disable your border, your input is jumping

Reply:
I thought , that otline it's someth other)))) but not like property . I didn't hear about it before this , my bad